packages/components/src/theme/tokens.ts

```diff
@@ -195,11 +195,11 @@ export const semantic = {
 
   fieldDisabled: `light-dark(${baseColors.gray10}, ${baseColors.gray180})`,
   /**
-   *  @deprecated since v10.2.0, please use the `fieldSkeleton01` instead.
+   *  @deprecated since v10.2.0, please use the `Skeleton01` instead.
    */
   fieldSkeleton: `light-dark(${baseColors.gray10}, ${baseColors.gray180})`,
-  fieldSkeleton01: `light-dark(${baseColors.gray10}, ${baseColors.gray180})`,
-  fieldSkeleton02: `light-dark(${baseColors.gray30}, ${baseColors.gray160})`,
+  Skeleton01: `light-dark(${baseColors.gray10}, ${baseColors.gray180})`,
```

Comment:
```suggestion
  skeleton01: `light-dark(${baseColors.gray10}, ${baseColors.gray180})`,
```

vi borde köra lowercase på tokens för att vara enhetliga

Reply:
fixat